app/views/layouts/3_column.html.erb

```diff
@@ -10,7 +10,7 @@
   <body>
    <!--<![endif]-->
 
-    <div id="theme" class="<%= yield(:theme) %>">
+    <div id="theme" class="<%= yield(:theme) %> new-font-theme">
```

Comment:
I believe the new OmniNav (v2) demo page layouts need to be updated with the `new-font-theme` class as well, right? On the Cascade Assets Dashboard, they're the pages under the third column, titled "OmniNav"

- `app/views/layouts/omninav_1col.html.erb`
- `app/views/layouts/omninav_2col.html.erb`
- `app/views/layouts/omninav_3col.html.erb`
- `app/views/layouts/omninav_home.html.erb`